hols/todo/infra/multi-cluster/resources/coredns-custom-c2.template.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2025 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# coredns-c1.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: coredns-custom
+  namespace: kube-system
+data:
+  c2.server: | # All custom server files must have a “.server” file extension.
+    cluster.c1:53 {
+      rewrite name substring cluster.c1 svc.cluster.local
+      # Change to c1 load balancer private ip
+      forward . 10.1.2.14
+    }
+#    cluster.c3:53 {
```

Comment:
Commented-out.

Reply:
keep format as reference in case a 3rd cluster is in play

Reply:
Add a comment about that. E.g. "To setup a 3rd cluster, un-comment the following section"

Reply:
done